criu/sockets.c

```diff
@@ -972,7 +972,7 @@ int collect_sockets(struct ns_id *ns)
 	req.r.i.sdiag_protocol = IPPROTO_ICMPV6;
 	req.r.i.idiag_ext = 0;
 	req.r.i.idiag_states = -1; /* All */
-	set_collect_bit(req.r.n.sdiag_family, req.r.n.sdiag_protocol);
+	set_collect_bit(req.r.i.sdiag_family, req.r.i.sdiag_protocol);
```

Comment:
It looks like this change is not directly related to enabling checkpoint/restore of `net.ipv4.ping_group_range`.
Would you be able to move it in a separate patch?